target/linux/ath79/dts/ar7241_tplink.dtsi

```diff
@@ -94,18 +94,17 @@
 	};
 };
 
-&eth0 {
+&eth0 {		// WAN port, initialized last as eth1
```

Comment:
These comments are a little unnecessary for me. All chips after AR7240 behave like this.
By the way, maybe many of the devices that are migrated to ath79 have the wrong MAC address.

Reply:
@DragonBluep About comments, I think it could stay, as it is not very intuitive that on AR7240 and similar systems eth numbering in dts and in openwrt after interfaces initialization is swapped (eth0 becomes eth1 and vice versa). Also this particular SoC behavior is not that widely documented.
@hauke What do you think? Should we leave comments or remove, for PR to get through?

Reply:
@hauke Thanks for update. I've changed comment syntax. Everything should be ok now.